app/_gateway_entities/license.md

```diff
@@ -22,8 +18,217 @@ schema:
+local filesystem to a directory in the Docker container, making the file visible
+from the container.
+
+## License expiration
+
+### Behavior
+
+Licenses expire at 00:00 on the date of expiration, relative to the time zone the machine is running in.
+
+Kong Manager displays a banner with a license expiration warning starting at 15 days before expiration.
+Expiration warnings also appear in [{{site.base_gateway}} logs.
+
+After the License expires, {{site.base_gateway}} behaves as follows:
+
+* Kong Manager and its configuration are accessible and may be changed, however any Enterprise-specific features become read-only.
+* The Admin API allows OSS features to continue working and configured {{site.ee_product_name}} features to continue operating in read-only mode.
+* Proxy traffic, including traffic using Enterprise plugins, continues to be processed as if the License hadn't expired.
+* Other Enterprise features aren't accessible.
+* There may be some Enterprise features that are still writable, but they may also change later, so don't rely on this behavior.
+
+The behavior of the different deployment modes is as follows:
+
+- **Traditional:** Nodes will be able to restart/scale as needed.
+- **Hybrid:** Existing data planes or new data planes **can accept** config from a control plane with an expired License.
+- **DB-less and KIC:** New nodes **cannot** come up, restarts will break.
+
+You can update your License with a `PUT` request to the [`/license/{license-id}` Admin API endpoint](/api/gateway/admin-ee/).
+
+### Logs
+
+{{site.base_gateway}} logs the License expiration date on the following schedule:
+* 90 days before: `WARN` log entry once a day
+* 30 days before: `ERR` log entry once a day
+* At and after expiration: `CRIT` log entry once a day
+
+## Monitor License usage and deployment information
+
+A license report contains information about your {{site.base_gateway}} database-backed deployment, including License usage and deployment information. The license report module manually generates a report containing usage and deployment data by sending a request to the `/license/report` endpoint. Share this information with Kong Support to perform a health-check analysis of product utilization and overall deployment performance to ensure your organization is optimized with the best License and deployment plan for your needs.
+
+The license report **does not**:
+*   Automatically generate a report or send any data to any Kong servers
+*   Track or generate any data other than the data that is returned in the response after you send a request to the endpoint
+
+{:.important}
+> **Important:** The license report functionality cannot be used in a DB-less deployment.
+
+### Get a license report
+
+| Method | Command |
+|--------|----------|
+| JSON | `curl -i -X GET http://localhost:8001/license/report` |
+| TAR | `curl http://localhost:8001/license/report -o response.json && tar -cf report-$(date +"%Y_%m_%d_%I_%M_%p").tar response.json` |
```

Comment:
```suggestion
| Method | Command |
|--------|----------|
| API | `curl -i -X GET http://localhost:8001/license/report` |

```

The second command dumps it from the same endpoint as the first and makes it into a tar file. I don't think this is needed at all. I would just say 'using the license/report' endpoint'